src/client/Microsoft.Identity.Client/Platforms/Android/AndroidPlatformProxy.cs

```diff
@@ -227,6 +229,11 @@ public override bool CanBrokerSupportSilentAuth()
             }
         }
 
+        public override bool IsBrokerInstalledAndInvokable()
+        {
+            return base.IsBrokerInstalledAndInvokable();
```

Comment:
base.isBrokerInstalledAndInvokable just returns false no?

Reply:
ah, yes, this is dead code. removing